draft-ietf-quic-applicability.md

```diff
@@ -487,6 +487,30 @@ Padding can also be used by an application to reduce leakage of
 information about the data that is sent. A QUIC implementation can expose an
 interface that allows an application layer to specify how to apply padding.
 
+# Error Handling
+
+QUIC recommends that endpoints that detect errors should signal the occurance to
+the peer. Errors can occur at the transport level and the application level.
+Transport errors, such as a protocol violation, affect the entire connection.
+Applications that use QUIC can define their own error detection and signalling
+(see, for example, {{Section 8 of QUIC-HTTP}}). Application errors can affect an
+entire connection or a single stream.
+
+QUIC defines an error code space that is used for error handling. QUIC
+encourages endpoints to use the most-specific code, although any applicable code
+is permitted including generic ones. Applications using QUIC can define an error
+code space that is independent from QUIC or other applications (see, for
+example, {{Section 8.1 of QUIC-HTTP}}). The values in an application error code
+space are reused across connection-level and stream-level errors.
```

Comment:
```suggestion
example, {{Section 8.1 of QUIC-HTTP}}). The values in an application error code
space can be reused across connection-level and stream-level errors.
```

This is not completely necessary, though this is what HTTP does.  They could be separate spaces if that was what people wanted.  That is, if 2^62 error codes is not enough, you can at least double the space.

Reply:
Agree, although this did kind of blindside me. I think QUIC transport's term "application protocol error code space" really reinforced my incorrect mental model.